cmake/download_onnxruntime.cmake

```diff
@@ -0,0 +1,25 @@
+if(DEFINED TRITON_ONNXRUNTIME_PACKAGE_URL)
+
+  set(DOWNLOAD_PATH "${CMAKE_BINARY_DIR}/_deps/downloads/onnxruntime.zip")
```

Comment:
How do we get the original `onnxruntime.zip`? Is that a separate (optional) build step in our pipeline?

Reply:
It's manual for now.